extensions/amp-subscriptions/0.1/amp-subscriptions.css

```diff
@@ -72,6 +72,10 @@ amp-subscriptions-dialog {
   transition: transform 0.3s ease-in;
 }
 
+amp-subscriptions-dialog.i-amphtml-subs-amp-story-dialog {
```

Comment:
Why not put this in the story subscriptions css?

Reply:
Ah true. Done. Thanks for the quick review!